lib/msal-core/src/ScopeSet.ts

```diff
@@ -141,15 +130,44 @@ export class ScopeSet {
      * Appends extraScopesToConsent if passed
      * @param {@link AuthenticationParameters}
      */
-    static appendScopes(reqScopes: Array<string>, reqExtraScopesToConsent: Array<string>): Array<string> {
-        if (reqScopes) {
-            const convertedExtraScopes = reqExtraScopesToConsent ? this.trimAndConvertArrayToLowerCase([...reqExtraScopesToConsent]) : null;
+    static appendScopes(reqScopes: Array<string>, scopesToAppend: Array<string>): Array<string> {
+        if(reqScopes) {
+            const convertedScopesToAppend = scopesToAppend ? this.trimAndConvertArrayToLowerCase([...scopesToAppend]) : null;
             const convertedReqScopes = this.trimAndConvertArrayToLowerCase([...reqScopes]);
-            return convertedExtraScopes ? [...convertedReqScopes, ...convertedExtraScopes] : convertedReqScopes;
+            return convertedScopesToAppend ? [...convertedReqScopes, ...convertedScopesToAppend] : convertedReqScopes;
         }
         return null;
     }
 
     // #endregion
 
+    /**
+     * Add openid and profile to scopes array
+     * OIDC required scopes: https://openid.net/specs/openid-connect-basic-1_0.html#Scopes
+     * @param scopes
+     */
+    static generateLoginScopes(scopes: Array<string>): Array<string> {
```

Comment:
Can we call this something like `addDefaultScopes` since this is no longer Login specific?

Reply:
How do you feel about `generateOIDCScopes`? I hesitate to call it `default` since they're only default for the request URL and not the request configuration (for response_type definition purposes they're not necessarily included).

Reply:
I like that, more specific. To stick to camel case convention: `generateOidcScopes` or maybe `addOpenIdScopes` if you don't like lowercasing the acronym

Reply:
Yeah, I'll also update the containsOIDCScopes method I added to match the camel case convention (containsOidcScopes)